op-guide/docker.md

```diff
@@ -0,0 +1,154 @@
+# 使用 Docker 构建和运行 TiDB
+
+本文档用于说明如何使用 Docker 部署 TiDB 服务
+
+## Docker 安装和环境准备
+
+#### 安装 Docker
+使用 Docker 可以非常快速搭建起一套 TiDB 环境。Docker 可以方便地在 Linux / Mac OS X / Windows 环境上安装，安装过程可参考 Docker 官网 https://docs.docker.com/linux/
+
+#### 准备环境
+完整的 TiDB 运行需要三个组件，TiDB, TiKV 和 PD。
+* tidb-server 是 TiDB 的执行进程，负责客户端的接入，对用户的 SQL 进行解析、优化和执行，分解成下层的 KV 操作，并将执行结果进行聚合返回给客户端。TiDB 对外兼容 MySQL 协议，可以直接使用 MySQL Client 进行测试
+* tikv-server，作为 TiDB 的分布式 KV 存储引擎，可以在网络互通的多机环境部署，使用 Raft 协议实现强一致性的数据复制，因此推荐 tikv-server 的部署节点为奇数个，当多数派节点存活 TiKV 保持可用状态
```

Comment:
This description is not exact,  we can guarantee that if the store count is more than `max-peer-count` config,  the TiKV is  high available.